common/src/main/java/com/gentics/mesh/router/route/FailureHandler.java

```diff
@@ -188,10 +188,9 @@ public void handle(RoutingContext rc) {
 				// That's why we don't reuse the error message here.
 				String msg = I18NUtil.get(ac, "error_internal");
 				rc.response().setStatusCode(500);
-				rc.response().end(JsonUtil.toJson(new GenericMessageResponse(msg), ac.isMinify(httpServerConfig)));
+				rc.response().end(JsonUtil.toJson(new ExceptionResponse(msg, topLevelFailure), ac.isMinify(httpServerConfig)));
```

Comment:
This will expose internal information (of the thrown exception) to the client. We don't want that. We want to have the topLevelFailure with its stacktrace to be logged on the server.